fast64_internal/oot/actor/operators.py

```diff
@@ -6,13 +6,55 @@
 from ..oot_constants import ootData
 
 
+class OOT_SearchChestContentEnumOperator(Operator):
+    bl_idname = "object.oot_search_chest_content_enum_operator"
+    bl_label = "Select Chest Content"
+    bl_property = "chestContent"
+    bl_options = {"REGISTER", "UNDO"}
+
+    chestContent: EnumProperty(items=ootData.actorData.ootEnumChestContent, default="item_heart")
+    objName: StringProperty()
+    propName: StringProperty()
```

Comment:
``propName`` is one of the properties created by fast64 when it loads (see ``initOOTActorProperties``), in this context it will be the name of the property holding the value of the chest item

Reply:
also small clarification here too, ``propName`` depends on the actor id and the prop index (as defined by the XML data ``ActorList.xml``), basically it's ``actor_id.propN`` where ``N`` is the index I mentioned

for these reasons you can't use an hardcoded value since we want to keep this whole system flexible enough so that people can edit the XML data to add their own actor or adjust an actor depending on the OoT mod they do (not mandatory of course)